runway/module/__init__.py

```diff
@@ -39,6 +40,15 @@ def generate_node_command(command, command_opts, path):
             cmd_list = [NPX_BIN,
                         '-c',
                         "%s %s" % (command, ' '.join(command_opts))]
+        # Python 3.8+ removes the silly need for redundant quoting
+        # https://docs.python.org/3/whatsnew/3.8.html#optimizations
+        elif platform.libc_ver()[1] and (
+                LooseVersion(platform.libc_ver()[1]) >= LooseVersion('2.24')) and (
```

Comment:
the linked article states that this effects macOS and Linux but this condition will never be truthy on macOS.

```shell
$ python -c "import platform; print(platform.libc_ver())"
('', '')
```

Reply:
It didn't occur to me yesterday, after having some trouble parsing the changelog, to check [the source](https://github.com/python/cpython/blob/v3.8.0/Lib/subprocess.py#L634-L678) to figure it out (I instead did dumb stuff like test on Alpine).

Can you confirm the change in behavior on macOS @ITProKyle ? Don't need to test the whole thing in runway with serverless, just trying to call subprocess on `npx -c 'foo bar'` on 3.8 vs <3.8. I could do it _but lazy_.

Reply:
tested with this script on macOS 10.15.4

```python
import subprocess

subprocess.check_call(['npx', '-c', "sls --version"])
subprocess.check_call(['npx', '-c', "''sls --version''"])
subprocess.check_call(['npx', '-c', "'sls --version'"])
```

with python version 2.7.15, 3.7.6, 3.8.1, and 3.8.2. all of which choke on the third `.check_call()` with a similar error.

```
/bin/sh: sls --version: command not found
Traceback (most recent call last):
  File "test.py", line 5, in <module>
    subprocess.check_call(['npx', '-c', "'sls --version'"])
  File "/Users/kyle/.pyenv/versions/3.8.2/Python.framework/Versions/3.8/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['npx', '-c', "'sls --version'"]' returned non-zero exit status 127.
```